packages/modal/src/components/Modal.tsx

```diff
@@ -42,22 +49,13 @@ class Modal extends React.Component<ModalProps, ModalState> {
     }
   }
 
-  el: HTMLDivElement;
-
   constructor(props: ModalProps) {
     super(props);
     this.el = document.createElement('div');
-    this.state = {
-      show: false,
-      visible: false
-    };
-
-    this.bindHandlers();
-  }
 
-  bindHandlers() {
```

Comment:
I'm removing this method because it will allow this method to be used publicly since JS don't have a real private method